notebook.js

```diff
@@ -48,8 +48,7 @@
     };
 
     var getAnsi = function () {
-        var req = condRequire("ansi_up");
-        var lib = root.ansi_up || req; 
+        var lib = root.ansi_up || condRequire("ansi_up");
```

Comment:
I don't see how this fixes or improves (other than condensing) the previous code. But maybe I'm overlooking something obvious. Can you explain?